src/components/shared/ResultList.js

```diff
@@ -38,24 +40,41 @@ export default class ResultList extends Component<Props> {
       key: `${item.id}${item.value}${index}`,
       ref: String(index),
       title: item.value,
+      "aria-labelledby": `${item.id}-title`,
+      "aria-describedby": `${item.id}-subtitle`,
+      role: "option",
       className: classnames("result-item", {
         selected: index === selected
       })
     };
 
     return (
       <li {...props}>
-        <div className="title">{item.title}</div>
-        <div className="subtitle">{item.subtitle}</div>
+        <div id={`${item.id}-title`} className="title">
+          {item.title}
+        </div>
+        <div id={`${item.id}-subtitle`} className="subtitle">
+          {item.subtitle}
+        </div>
       </li>
     );
   }
 
   render() {
-    const { size, items } = this.props;
+    const { size, items, role, selected } = this.props;
 
     return (
-      <ul className={classnames("result-list", size)}>
+      <ul
+        className={classnames("result-list", size)}
+        role={role}
+        aria-activedescendant={
```

Comment:
Nice

Reply:
Hi @jasonLaster , is there anything we can do or should we just wait for our pull request to be reviewed by others?

Reply:
I was doing some more research on how best to implement accessible autocomplete and I was hoping it's OK I ask for a couple of changes (from my original comments), here in particular:

aria-activedescendant should belong to the input element instead of the results list.

Reply:
Hi, @yzen. We are trying to move aria-activedescendant to the input field in SearchInput.js, but we are having trouble accessing the items list and the selected items index from ResultList.js. 

We have tried importing ResultList in SearchInput but I think the issue is that the components in  ResultList have not yet loaded when we try to access them in SearchInput.

One of my team members has also looked into passing the onSelectResultItem function from QuickOpenModal.js to SearchInput.js. 

So far, neither have worked.